lib/emojibot/bot.ex

```diff
@@ -4,7 +4,7 @@ defmodule Emojibot.Bot do
 
   def handle_event(%{name: name, type: "emoji_changed", subtype: "add"}, slack, state) do
     emoji_channel = Application.get_env(:emojibot, :emoji_channel)
-    send_message(":#{name}:", emoji_channel, slack)
+    send_message(":#{name}: (name)", emoji_channel, slack)
```

Comment:
Since the emoji name is kind of like a "code", I’d suggest:

```suggestion
    send_message(":#{name}: (`name`)", emoji_channel, slack)
```

What do you think?